src/utils/objects.ts

```diff
@@ -0,0 +1,6 @@
+export const isEmpty = (value: any) =>
+  value === null ||
+  value === '' ||
+  Number.isNaN(value) ||
+  (value instanceof Object && Object.entries(value).length === 0) ||
+  (Array.isArray(value) && value.length === 0)
```

Comment:
I wouldn't go for import * as Objects, but rather import {Objects} from 'src/utls'

here you can
```
const isEmpty = (value: any) =>
   value === null ||
   value === '' ||
   Number.isNaN(value) ||
   (value instanceof Object && Object.entries(value).length === 0) ||
   (Array.isArray(value) && value.length === 0)

export const Objects = {
  isEmpty
}
```

in utils index:
```
export {Objects} from './objects'
```

Reply:
right, it will be easier to import/use it elsewhere in this way...